README.md

```diff
@@ -2,30 +2,90 @@
 
-# POLAR 🗺️
+<h1 align="center"><img alt="POLAR" height="80px" src="./pages/assets/iceberg.svg" /></h1>
 
-The **Plugins for OpenLAyeRs** library is based on the [masterportalAPI](https://bitbucket.org/geowerkstatt-hamburg/masterportalapi) and [OpenLayers](https://openlayers.org/).
+**Plugins for OpenLAyeRs** is based on the [masterportalAPI](https://bitbucket.org/geowerkstatt-hamburg/masterportalapi) and [OpenLayers](https://openlayers.org/).
 
-## The Idea 💡
+POLAR is ...
 
-POLAR is built to ease the creation of new map clients: A lot of feature requests in map clients are recurring and can be fulfilled with reusable parts. Then again, many map clients require a _little extra_.
+* ... a configurable map client package.
+* ... a flexible map client factory.
+* ... an extensible library.
 
-POLAR is built to serve both worlds. For generic use cases, generic clients are ready-made and usable by configuration. More specific use cases can be matched with special clients that still make use of the plugins, but fill in the missing parts right where they belong.
+## Quick Start
 
-POLAR runs both as full page application and as component. The most common usage is as component: Think of it as a form input where the input data is geospatial.
+Usage without NPM documented in chapter "Getting started (for developers)".
+
+```bash
+npm i @polar/client-generic
+```
+
+```js
+import polar from '@polar/client-generic'
+
+polar.createMap({
+  // a div must have this id
+  containerId: 'polarstern',
+  // any service register – this is Hamburg's
+  services: 'https://geodienste.hamburg.de/services-internet.json',
+  mapConfiguration: {
+    // this initially shows Hamburg's city plan
+    layers: [{
+      id: '453',
+      visibility: true,
+      type: 'background',
+    }]
+  }
+})
+```
+
+See our [documentation page](https://dataport.github.io/polar/) for all features and configuration options included in this modulith client, with running examples.
 
-## Maps 🗺️
+## Example clients
 
 The most common use case for this client is in citizen's application processes regarding public service.
 
-Other clients with more specific code (and more visibility) include the [Denkmalkarte Schleswig-Holstein](https://efi2.schleswig-holstein.de/dish/dish_client/index.html), a memorial map, and the [Meldemichel Hamburg](https://static.hamburg.de/kartenclient/prod/), a map to inspect and create reports regarding damages to public infrastructure. The latter is currently being migrated to the version seen in this repository.
+Other clients with more specific code include the [Denkmalkarte Schleswig-Holstein](https://efi2.schleswig-holstein.de/dish/dish_client/index.html), a memorial map, and the [Meldemichel Hamburg](https://static.hamburg.de/kartenclient/prod/), a map to inspect and create reports regarding damages to public infrastructure. The latter is currently being migrated to the version seen in this repository.
```

Comment:
Using the Eigenname seems fitting here, especially since we have an explanation in the subsequent subordinate clause; we're not translating Meldemichel as "Reporting Michael" either.